lib/deimos/config/configuration.rb

```diff
@@ -90,7 +90,9 @@ def self.configure_producer_or_consumer(kafka_config)
       if kafka_config.respond_to?(:bulk_import_id_column) # consumer
         klass.config.merge!(
           bulk_import_id_column: kafka_config.bulk_import_id_column,
-          replace_associations: kafka_config.replace_associations
+          replace_associations: kafka_config.replace_associations,
+          bulk_import_id_generator: kafka_config.bulk_import_id_generator ||
+            Deimos.config.consumers.bulk_import_id_generator
```

Comment:
Do we have a use case for a global setting as well as a per-consumer setting? I feel like this would have to be something specified per consumer.

Reply:
Just for convenience. All of of our consumers use the same batch generator (Ulids)